website/pages/blog/product-updates-1.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "CloudQuery Product Updates #1"
+tag: product
+date: 2022/11/01
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly product updates, and starting from #1 as we figured it is easier to use numbers than months :)
+
+## Sources
```

Comment:
I think it would be cool if we could highlight the codegen improvements that allow us to automatically regenerate all (AWS) resources so that we are able to keep up to date with AWS releases on launch day